src/app/components/dropdown-page/dropdown-page.component.html

```diff
@@ -247,4 +247,57 @@ <h1 style="padding: 10px;">Hello!</h1>
       </ngx-dropdown>
       ]]>
   </ngx-codemirror>
+</ngx-section>
+
+<ngx-section class="shadow" [sectionTitle]="'Show Events'">
```

Comment:
when using static inputs in all the areas below you do not need to bind it using `[input]="'value'"` you can just use `input="value"` since you know it isn't going to change

Reply:
we have plans to update the demo pages, just haven't gotten to it yet.

Reply:
I think for the code samples we should keep the square brackets... to communicate that this is a component input.  Elsewhere we can switch.

Reply:
We are going to be adding an API tab to communicate inputs/outputs to the users.